File: CTDL1/New folder/Common7/IDE/ItemTemplates/CSharp/1033/XamlWFStateMachineService/IWorkflow.cs
using System;
using System.Linq;
using System.Collections.Generic;
using System.Text;
using System.ServiceModel;

namespace $rootnamespace$
{
	// NOTE: You can use the "Rename" command on the "Refactor" menu to change the interface name "$safeitemrootname$" in both code and config file together.
	[ServiceContract]
	public interface $safeitemrootname$
	{

		[OperationContract]
		string GetData(int value);

	}
}

File: CTDL1/New folder/Common7/IDE/ItemTemplatesCache/VisualBasic/1033/XamlWFSequentialService/IWorkflow.vb
Imports System.ServiceModel

' NOTE: You can use the "Rename" command on the context menu to change the interface name "$safeitemrootname$" in both code and config file together.
<ServiceContract()> _
Public Interface $safeitemrootname$

    <OperationContract()> _
    Function GetData(ByVal value As Integer) As String

End Interface
